cds-plugin.js

```diff
@@ -64,7 +48,10 @@ cds.once("served", async () => {
                         const actionName = boundAction.name.split('.').pop();
 
                         // Register for print related handling
-                        srv.after(actionName, print);
+                        srv.after(actionName, async (results, req) => {
+                            const printer = await cds.connect.to("print");
+                            return printer.print(req);
+                        });
```

Comment:
```suggestion
                          const printer = await cds.connect.to("print");
                        srv.after(actionName, async (results, req) => {
                            return printer.print(req);
                        });
```

You can connect to the service outside of the action handler, as otherwise each print request will establish a new connection which will increase the req time